Sources/SkyFloatingLabelTextFieldWithIcon.swift

```diff
@@ -12,9 +12,49 @@
 import UIKit
 
 /**
+ Identify the type of icon. 
+    - font: Set your icon by setting the font of iconLabel
+    - image: Set your icon by setting the image of iconImageView
+ */
+public enum IconType: Int {
+    case font
+    case image
+}
+
+/**
  A beautiful and flexible textfield implementation with support for icon, title label, error message and placeholder.
  */
 open class SkyFloatingLabelTextFieldWithIcon: SkyFloatingLabelTextField {
+    
+    @IBInspectable
+    var _iconType: Int { // swiftlint:disable:this identifier_name
```

Comment:
I'm not crazy about this option either come to think of it. Can we call it `iconTypeValue` to indicate it's a value representation of `iconType`. Also open to any suggestions you might have too